core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,796 @@
+    <div class="oppia-section-six oppia-about-fixed-section">
+      <h2 class="oppia-about-outcome-title-mobile oppia-about-page-title-mobile" [innerHTML]="'I18N_ABOUT_PAGE_EASILY_CREATE_LESSON' | translate"></h2>
+      <div class="oppia-about-section-six-image-left">
+        <picture class="oppia-about-book-image">
+          <source type="image/webp" [srcset]="getStaticImageUrl('/about/books_dsk.webp')">
+          <source type="image/png" [srcset]="getStaticImageUrl('/about/books_dsk.png')">
+          <img [src]="getStaticImageUrl('/about/books_dsk.png')" alt="book image">
+        </picture>
+      </div>
+      <div class="oppia-about-section-six-right">
+        <h2 class="oppia-about-outcome-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_EASILY_CREATE_LESSON' | translate"></h2>
+        <p class="oppia-about-outcome-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_CREATE_LESSON_CONTENT' | translate"></p>
+        <div class="oppia-about-classroom-button-container">
+          <button type="submit" (click)="onClickCreateLessonButton()"
+                  class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_CREATE_LESSON_BUTTON' | translate">
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-seven oppia-about-section">
+    <h2 class="oppia-get-started-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_SECTION_SEVEN_TITLE' | translate"></h2>
+    <div class="oppia-about-classroom-button-container-three">
+      <button type="submit" (click)="onClickVisitClassroomButton()"
+              class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+              [innerHTML]="'I18N_ACTION_EXPLORE_LESSONS' | translate">
+      </button>
+    </div>
   </div>
 </div>
+
 <style>
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor {
-    background-color: #00645c;
-    border-radius: 4px;
-    color: white;
-    letter-spacing: 0.7px;
-    width: 33%;
-  }
-  .oppia-about-page-container .oppia-about-foundation-tab-anchor:hover {
-    background-color: #3d9991;
-    text-decoration: none;
-  }
-  .oppia-about-page-container .oppia-about-text-align {
-    text-align: center;
+  .oppia-about-section {
+  background-color: #fff;
+  clear: both;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  max-width: 100%;
+  width: 100vw;
+}
+.oppia-about-fixed-section {
+  justify-content: center;
+  width: 1440px;
+}
+.oppia-about-page-section-one {
+  background-color: #F4F4F4;
+  height: 60vh;
+  padding-top: 0;
+}
+.oppia-about-page-section-one .oppia-about-page-section-one-right {
+  float: right;
+  height: 60vh;
+  width: 50%;
+}
+.oppia-about-page-section-one .oppia-about-page-section-one-left {
+  float: right;
```

Comment:
we are expecting it to float right in div oppia-about-page-section-one-left which means it will be side of the image and have added margin to the content to give space b/w image and content. (we have done this because we have set a fixed width of sections so that they don't spread in very large size devices.)

Reply:
Can we use `float-right`, `float-left` (from bootstrap) classes instead of defining a new class with the same style?